examples/sol-anchor-contract/README.md

```diff
@@ -0,0 +1,28 @@
+# Pyth SDK Example Program for Solana and the Anchor Library
+
+This example implements the same functionalities as the `sol-contract` example.
+The difference is that this example uses the `anchor` library while the `sol-contract` example uses the lower-level solana interface.
+Please refer to the README of `sol-contract` for a description of the functionalities.
+
+## Run this program
+We assume that you have installed `anchor`, `npm` and `yarn`.
+
+```shell
+# Enter the directory and build this example
+> cd examples/sol-contract-anchor
+> anchor build
+
+# Config the solana CLI by setting its url to devnet
+> solana config set --url https://api.devnet.solana.com
```

Comment:
No need to do this. `anchor` gets the rpc url from the field `provider.cluster` in `Anchor.toml`. So you don't need to set it in the solana cli.